src/main/resources/plugin.yml

```diff
@@ -8,17 +8,24 @@ description: Warp stones for survival
 website: https://atrius.xyz
 commands:
   waystones:
-    description: Main command
-    usage: /waystones
+    description: Displays plugin info and usage
+    usage: /waystones [command]
+    aliases: [ws]
 permissions:
   waystones.*:
     description: Gives access to all Waystones commands
     children:
       waystones.reload: true
-      waystones.warpkey: true
-  waystones.reload:
-    description: Allows reloading the plugins config file
+      waystones.getkey.self: true
+      waystones.getkey.others: true
+  waystones.getkey.*:
+    description: Allows giving warpkeys to all players
+    children:
+      waystones.getkey.self: true
+      waystones.getkey.others: true
+  waystones.getkey.self:
+    description: Allows giving WarpKeys to yourself
     default: op
-  waystones.warpkey:
-    description: Allows giving warpkeys to self and players
+  waystones.getkey.others:
```

Comment:
It could, but that'd make it equivalent to `getkey.*`. I'm not sure why anyone would need the `.others` perm exclusively either, but if they do I figured it's there.

I should be able to rename `waystones.getkey.*` to `waystones.getkey`, omitting the asterisk, and that'd maybe make it a bit clearer.

Reply:
Maybe a better way to list the perms is `waystones.getkey.self` and `waystones.getkey.all`?

Reply:
That could work, I feel we'll likely come back and revisit permission names once more permissions and commands are implemented, but I'm happy with leaving them under `waystones.getkey.self` and `waystones.getkey.all` as of now.